manifests/module/registration.pp

```diff
@@ -0,0 +1,17 @@
+# @summary The built in Registration module
+#
+# @param enabled
+#  Whether the module is enabled or disabled.
+#
+# @param listen_on
+#   Where to listen on.
+class foreman_proxy::module::registration (
+  Boolean $enabled = $foreman_proxy::registration,
+  Foreman_proxy::ListenOn $listen_on = $foreman_proxy::registration_listen_on,
+) {
+  foreman_proxy::module { 'registration':
+    enabled   => $enabled,
+    feature   => 'Registration',
```

Comment:
Note to self: confirmed that this matches Foreman's feature name (https://github.com/theforeman/foreman/commit/31b0d198080ca9f1444308f726870f0bf49208ec)